cmake/FindPAPI.cmake

```diff
@@ -0,0 +1,101 @@
+#[=======================================================================[.rst:
+FindPAPI
+--------
+
+Find the native PAPI headers and libraries.
+
+IMPORTED Targets
+^^^^^^^^^^^^^^^^
+
+This module defines :prop_tgt:`IMPORTED` target ``PAPI::libpapi``, if PAPI has been found.
+
+Result Variables
+^^^^^^^^^^^^^^^^
+
+This module defines the following variables:
+
+``PAPI_FOUND``
+  "True" if ``papi`` found.
+
+``PAPI_INCLUDE_DIR``
+  where to find ``papi``/``papi.h``, etc.
+
+``PAPI_LIBRARY``
+  List of libraries when using ``papi``.
+
+``PAPI_VERSION_STRING``
+  The version of ``papi`` found.
+
+This module defines ``PAPI::libpapi`` target for PAPI library.
+
+#]=======================================================================]
+
+# Look for the header file.
+if(NOT PAPI_INCLUDE_DIR)
+  find_path(
+    PAPI_INCLUDE_DIR
+    NAMES papi.h
+    HINTS /usr/include /usr/local/include)
+  mark_as_advanced(PAPI_INCLUDE_DIR)
+endif()
```

Comment:
```suggestion
```

Reply:
Suggestion applied in b20828a09913424257c98cf33491de3199feb4f2